contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -71,78 +90,156 @@ parameters:
       - input: dstIP
         output: dstLocation
         type: add_location
-    type: network
-- extract:
+- name: extract_conntrack
+  extract:
+    type: conntrack
+    conntrack:
+      keyDefinition:
+        fieldGroups:
+        - name: src
+          fields:
+          - srcIP
+          - srcPort
+        - name: dst
+          fields:
+          - dstIP
+          - dstPort
+        - name: protocol
+          fields:
+          - proto
+        hash:
+          fieldGroupRefs:
+          - protocol
+          fieldGroupARef: src
+          fieldGroupBRef: dst
+      outputRecordTypes:
+      - newConnection
```

Comment:
Do we describe anywhere how to use these fields? in the README?

Reply:
Not yet